File: OOPs/Collections/ArrayBasedImplementation/Driver/Driver.cs
using System;



namespace OOPs.Collections.ArrayBaseImplementation
{
	public class Driver
	{
		public static void Main(string[] args)
		{
			CustomArrayList<string> customArrayList = new CustomArrayList<string>();
		
			customArrayList.Add("Milk");
			customArrayList.Add("Bread");
			customArrayList.Add("Honey");
			customArrayList.Add("Coffee");
			customArrayList.Add("Pen");
			customArrayList.Insert(2,"Beer");
			
			customArrayList.print();
			customArrayList.Clear();
			customArrayList.print();
			

		}
	}
}

File: Java/Data Structures and Algorithms/LinkedList/src/LinkedList/driver/Driver.java
package LinkedList.driver;
import LinkedList.singlyLinkedList.SinglyLinkedList;

class Driver
{
	public static void main(String[] args)
	{
		SinglyLinkedList sLL = new SinglyLinkedList();
		
		sLL.insert(10);
		sLL.insert(20);
		sLL.insert(30);
		sLL.insert(40);
		sLL.insert(50);
		
		sLL.print();

		
	}
}